src/sentry/utils/otp.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import absolute_import
+
+import time
+import hmac
+import random
+import base64
+import qrcode
+import urllib
+import hashlib
+from datetime import datetime
+
+from sentry.utils.dates import to_timestamp
+
+from django.utils.crypto import constant_time_compare
+
+
+def generate_secret_key(length=32):
```

Comment:
Django has `from django.utils.crypto import get_random_string` that does just this as well.

So could be replaced with:

```python
return get_random_string(length, 'ABCDEFGHIJKLMNOPQRSTUVWXYZ234567')
```